pom.xml

```diff
@@ -7,7 +7,7 @@
 
 	<groupId>io.apimatic</groupId>
 	<artifactId>core-interfaces</artifactId>
-	<version>0.1.3</version>
+	<version>0.1.4</version>
```

Comment:
No need to upgrade this version as `0.1.3` has not been released yet so this change will become a part of `0.1.3` now.

Reply:
done